src/index.js

```diff
@@ -28,7 +28,8 @@ class ScrollableList extends Component {
   componentWillReceiveProps(props) {
     if (this.props.data !== props.data) {
       this.setState({
-        dataSource: this.ds.cloneWithRows(props.data),
+        // If props.data == null, set an empty array instead
```

Comment:
In fact, it will use an empty array whenever `props.data` is _falsy_ like `undefined`, `null`, `void 0`, etc. Make sure the comment is not misleading (I'd rather delete it!). Everything else looks 👌